<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditView</class>
 <widget class="QDialog" name="CreditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>335</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>523</width>
    <height>335</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>523</width>
    <height>335</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CreditView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
        background-color: rgba(93, 92, 94, 209);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_calcCredit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font:14pt &quot;.AppleSystemUIFont&quot;;
	color: rgb(255, 255, 255);
     border: 1px solid grey;
	background-color: rgba(93, 92, 94, 209);
}
QPushButton:pressed {
  background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                             stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_diff">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
font:8pt &quot;.AppleSystemUIFont&quot;;
	color: rgb(255, 255, 255);
     border: 1px solid grey;
	background-color: rgba(93, 92, 94, 209);
}</string>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>50</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font:10pt &quot;.AppleSystemUIFont&quot;;
	color: rgb(255, 255, 255);
     border: 1px solid grey;
	background-color: rgba(93, 92, 94, 209);
}</string>
   </property>
   <property name="text">
    <string>% в год</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_period">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font:10pt &quot;.AppleSystemUIFont&quot;;
	color: rgb(255, 255, 255);
     border: 1px solid grey;
	background-color: rgba(93, 92, 94, 209);
}</string>
   </property>
   <property name="text">
    <string>рублей</string>
   </property>
   <property name="buddy">
    <cstring>label_5</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_month">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font:10pt &quot;.AppleSystemUIFont&quot;;
	color: rgb(255, 255, 255);
     border: 1px solid grey;
	background-color: rgba(93, 92, 94, 209);
}</string>
   </property>
   <property name="text">
    <string>месяцев</string>
   </property>
   <property name="buddy">
    <cstring>label_5</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bet">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
   border: 1px solid grey;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_summ">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_annuit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
font:8pt &quot;.AppleSystemUIFont&quot;;
	color: rgb(255, 255, 255);
     border: 1px solid grey;
	background-color: rgba(93, 92, 94, 209);
}</string>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_period">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font:10pt &quot;.AppleSystemUIFont&quot;;
	color: rgb(255, 255, 255);
     border: 1px solid grey;
	background-color: rgba(93, 92, 94, 209);
}</string>
   </property>
   <property name="text">
    <string>Срок кредита (месяцев)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font:10pt &quot;.AppleSystemUIFont&quot;;
	color: rgb(255, 255, 255);
     border: 1px solid grey;
	background-color: rgba(93, 92, 94, 209);
}</string>
   </property>
   <property name="text">
    <string>Ставка (% в год)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font:10pt &quot;.AppleSystemUIFont&quot;;
	color: rgb(255, 255, 255);
     border: 1px solid grey;
	background-color: rgba(93, 92, 94, 209);
}</string>
   </property>
   <property name="text">
    <string>Вид кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_summ">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font:10pt &quot;.AppleSystemUIFont&quot;;
	color: rgb(255, 255, 255);
     border: 1px solid grey;
	background-color: rgba(93, 92, 94, 209);
}</string>
   </property>
   <property name="text">
    <string>Сумма кредита (рублей)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_over_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    border: 1px solid grey;
	background-color: rgb(155, 155, 155);
	font: 10pt &quot;Arial&quot;;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font:10pt &quot;.AppleSystemUIFont&quot;;
	color: rgb(255, 255, 255);
     border: 1px solid grey;
	background-color: rgba(93, 92, 94, 209);
}</string>
   </property>
   <property name="text">
    <string>Общая выплата</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    border: 1px solid grey;
	background-color: rgb(155,155,155);
	font: 10pt &quot;Arial&quot;;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_over">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font:10pt &quot;.AppleSystemUIFont&quot;;
	color: rgb(255, 255, 255);
     border: 1px solid grey;
	background-color: rgba(93, 92, 94, 209);
}</string>
   </property>
   <property name="text">
    <string>Переплата по кредиту</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthly_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>155</red>
         <green>155</green>
         <blue>155</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    border: 1px solid grey;
	background-color: rgb(155, 155,155);
	font: 10pt &quot;Arial&quot;;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthly">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font:10pt &quot;.AppleSystemUIFont&quot;;
	color: rgb(255, 255, 255);
     border: 1px solid grey;
	background-color: rgba(93, 92, 94, 209);
}</string>
   </property>
   <property name="text">
    <string>Ежемесячный платеж</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_closecredit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <zorder>pushButton_calcCredit</zorder>
  <zorder>label_6</zorder>
  <zorder>label_5</zorder>
  <zorder>label_month</zorder>
  <zorder>lineEdit_bet</zorder>
  <zorder>lineEdit_summ</zorder>
  <zorder>radioButton_annuit</zorder>
  <zorder>label_period</zorder>
  <zorder>label_bet</zorder>
  <zorder>label_type</zorder>
  <zorder>label_summ</zorder>
  <zorder>label_over_2</zorder>
  <zorder>label_total</zorder>
  <zorder>label_total_2</zorder>
  <zorder>label_over</zorder>
  <zorder>label_monthly_2</zorder>
  <zorder>label_monthly</zorder>
  <zorder>lineEdit_period</zorder>
  <zorder>radioButton_diff</zorder>
  <zorder>pushButton_closecredit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
